src/data/faq.json

```diff
@@ -3245,11 +3245,21 @@
                     "anchor": "backup",
                     "description": "Backups refer to versions of a project at a certain point in time in the past, which can later be used to restore a project to its previous state."
                 },
+                {
+                    "term": "Basic Immunization",
+                    "anchor": "basic_immunization",
+                    "description": "All people who have undergone the initial vaccination series are considered to be primed. In the case of SARS-CoV-2, this generally comprises two vaccine doses, but only a single vaccine dose for those who have been proven to have recovered. Individuals who have already received additional doses of vaccination are also counted here. Complete basic immunization is not equivalent to complete vaccine protection."

```

Comment:
"to be primed" sounds strange in this context. See https://dictionary.cambridge.org/dictionary/english/prime (verb)

suggest as replacement:
"to have completed basic immunization"